Code/Framework/AzToolsFramework/AzToolsFramework/ViewportUi/ViewportUiDisplay.cpp

```diff
@@ -164,6 +166,28 @@ namespace AzToolsFramework::ViewportUi::Internal
         }
     }
 
+    const bool ViewportUiDisplay::HideSwitcher(ViewportUiElementId switcherId)
+    {
+        if (auto switcher = qobject_cast<ViewportUiSwitcher*>(GetViewportUiElement(switcherId).get()))
+        {
+            bool hidden = switcher->HideSwitcher();
```

Comment:
suggestion: Might as well just `return switcher->HideSwitcher();` (same below for show)